### ISSUE TITLE
Early Stopping/Patience Epochs

### ISSUE THREAD

**Issue by FarhAnonymous** [Peterande/D-FINE#184]
Any idea how to automatically stop training if your validation metric (AP) doesn't improve for patience epochs, saving compute resources and preventing overfitting?

**Comment by NielsRogge** [comment]
You can use the 🤗 model implementation (#276) and then use early stopping with either the 🤗 Trainer API, PyTorch Lightning or native PyTorch.
